TYPO3.Neos/Resources/Private/Translations/en/Modules.xlf

```diff
@@ -137,8 +137,8 @@
 			<trans-unit id="workspaces.publishAllChangesTo" xml:space="preserve">
 				<source>Publish all changes to {0}</source>
 			</trans-unit>
-			<trans-unit id="workspaces.legendCaption" xml:space="preserve">
-				<source>Legend</source>
+			<trans-unit id="changedContentCaption" xml:space="preserve">
```

Comment:
This would be one of the few changes that have no prefix (workspaces) does that make sense? Shouldn't it have one?